app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -161,6 +166,11 @@ public void onTracksInfoChanged(TracksInfo tracksInfo) {
         });
     }
 
+    public void subscribe(PlaybackControllerNotifiable notifier){
```

Comment:
Unknown nullability; explicitly declare as @Nullable or @NonNull to improve Kotlin interoperability

```suggestion
    public void subscribe(@NonNull PlaybackControllerNotifiable notifier) {
```